libr/core/cconfig.c

```diff
@@ -1233,6 +1233,20 @@ static bool cb_cfgcharset(void *user, void *data) {
 		char *syscs = r_str_newf ("%s%s.sdb", cs, cf);
 		if (r_file_exists (syscs)) {
 			rc = r_charset_open (core->print->charset, syscs);
+
```

Comment:
shouldnt this code be moved into r_charset_open() instead ? i dont see why it should be done in the config callback

Reply:
move the dbg_char_to_hex generation code inside the r_charset_open() function

Reply:
see this  ^